src/Trixi.jl

```diff
@@ -70,6 +70,7 @@ using Triangulate: Triangulate, TriangulateIO, triangulate
 export TriangulateIO # for type parameter in DGMultiMesh
 using TriplotBase: TriplotBase
 using TriplotRecipes: DGTriPseudocolor
+@reexport using TrixiBase: TrixiBase, trixi_include
```

Comment:
Sorry, I know I'm a little late to the party, but why do we export `TrixiBase`? Wouldn't it suffice to just export `trixi_include`?

Reply:
Uhm, yeah, I guess so. I think I only re-exported `trixi_include`, and later added `TrixiBase` because it's needed in Trixi somewhere, but I didn't notice that I added it to the re-export line.

Reply:
I suggest we change that - just in case - before creating a Trixi.jl release, such that we do not make TrixiBase.jl part of our public API.

Reply:
#1846